crates/holochain_conductor_api/src/config/conductor.rs

```diff
@@ -33,7 +33,7 @@
 //!   ## to what you can see here:
 //!   webrtc_config: {
 //!     "iceServers": [
-//!       { "urls": ["stun:devtest-stun-1.holochain.org:443"] }
+//!       { "urls": ["stun://stun.l.google.com:19302"] }
```

Comment:
I vote we go with the canonical

```suggestion
//!       { "urls": ["stun:stun.l.google.com:19302"] }
```

Reply:
I see that's not how its written in kitsune2, but I suppose that doesn't matter. https://github.com/holochain/kitsune2/blob/6bba29ab3566be90fb1fb496cf9f1af05368ba63/crates/kitsune2_showcase/src/app.rs#L96